packages/itwinui-react/src/core/Popover/Popover.tsx

```diff
@@ -262,6 +277,14 @@ export const Popover = React.forwardRef((props, forwardedRef) => {
   const triggerId = `${useId()}-trigger`;
   const hasAriaLabel = !!props['aria-labelledby'] || !!props['aria-label'];
 
+  useIsomorphicLayoutEffect(() => {
+    if (!positionReference) {
+      return;
+    }
+    popover.refs.setPositionReference(positionReference);
+    return () => void popover.refs.setPositionReference(null);
```

Comment:
Why is `void` needed here?

Reply:
`void` is syntactic sugar for wrapping with `{}`. it discards the return value (if any).
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void#non-leaking_arrow_functions